docs/governance/steering/charter.md

```diff
@@ -0,0 +1,181 @@
+* Delegate ownership of, responsibility for, and authority over areas of the project to specific entities<sup>[2](#footnote2)</sup>.
+* Charter and refine policy for defining new community groups<sup>[3](#footnote3)</sup>, and establish transparency and accountability policies for such groups.
+* Appoint leadership for chartered community groups.
+* Enforce charters for community groups<sup>[3](#footnote3)</sup> to ensure ongoing positive standing, or decomission any groups not in good standing.
+* Define and evolve project and group<sup>[3](#footnote3)</sup> governance structures and policies<sup>[4](#footnote4)</sup>.
+* Act as a final non-technical escalation point for any CCC community group or technical effort.
+* Decide who is a member of standing on the CCC project, and what privileges that entails.
+* Control and delegate access to and establish processes regarding project resources<sup>[5](#footnote5)</sup>.
+* Coordinate with FINOS regarding usage of the CCC brand and deciding which things can be called "Common Cloud Controls," as well as how that mark can be used in relation to other efforts or vendors.
+* Request funds and other support from FINOS (e.g. marketing, press, etc).
+* Interface with the FINOS Staff point of contact as needed for guidance and accountability.
+
+## Non-responsibilities of the Steering Committee
+
+* Technical decision-making.
+* Chartering, approval, or oversight of any sub-groups created by Steering-chartered community groups (those which are authorized to do so within their charter).
+* Files and resources outside of:
+  - the CCC project README.md
+  - Steering Committee governance documentation
+
+### FINOS Point of Contact
+
+In addition to general foundation support given to projects, the Steering Committee will have an assigned point of contact. This should be tracked in the same documentation as the list of current Steering Commmittee members.
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members and shall always be considered this size regardless of temporary vacancies, for the purpose of voting majority and supermajority considerations.
+
+**Reserved Seats —** Four (4) seats on the committee are reserved for FINOS members from financial services institutions. These seats may not be occupied by more than one individual from the same institution<sup>[6](#footnote6)</sup>. In the event that a committee member occupying an FSI seat undergoes a change of employment, they may retain their seat while in noncompliance for 30 calendar days before it is automatically considered vacant.
+
+**Community Seats —** Three (3) seats on the committee are not reserved, and may be filled by any community member from any organization.
+
+### Elections
+
+Every year, the Steering Committee holds a general election for open seats.
+
+Our [election policy] document covers the details for how this works.
```

Comment:
Broken election policy link

Reply:
Thanks Adrian— Per discussion with @robmoffat, we're not entirely sure whether this is an issue with the markdown or the preview. We'll merge with broken links as these are non-impactful to the governance proposal, and fix the links in a follow-up PR.